tools/ci_build/github/azure-pipelines/orttraining-1p-linux-gpu-ci-pipeline.yml

```diff
@@ -0,0 +1,69 @@
+parameters:
+  - name: build_flag
+    displayName: Build Flag
+    type: string
+    default: --enable_onnx_tests
+  - name: image_tag
+    displayName: Image Tag
+    type: string
+    default: latest
+
+stages:
+- stage: Python_Packaging
+
+  variables:
+    docker_image_prefix: onnxruntime-training
+    linux_gpu_dockerfile: ../../../../../dockerfiles/Dockerfile.training
+    python.version: 3.6
+    BuildCommand: >
+      tools/ci_build/github/linux/run_dockerbuild.sh
+      -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory)
+      -x "
+      --enable_training
+      --config $(buildConfig)
+      --skip_onnx_tests
+      --build_wheel
+      --enable_nvtx_profile
+      "
+  jobs:
+  - job: Linux_py_GPU_Wheels
```

Comment:
this job name is probably not that fitting anymore

Reply:
Changed job name